src/http/server/context.cr

```diff
@@ -7,8 +7,11 @@ class HTTP::Server
     # The `HTTP::Server::Response` to configure and write to.
     getter response : Response
 
+    # The `Socket::IPAddress` of the remote
+    getter remote_address : Socket::IPAddress
+
     # :nodoc:
-    def initialize(@request : Request, @response : Response)
+    def initialize(@request : Request, @response : Response, @remote_address : Socket::IPAddress = Socket::IPAddress.new("127.0.0.1", 3000))
```

Comment:
There should be no default. It's misleading.

Furthermore, it's not necessarily an IP address, it can be an UNIX address.